### ISSUE TITLE
Issue with table representation inside terminal.

### ISSUE THREAD

**Issue by ThisIsTeddyBear** [zayKenyon/VALORANT-rank-yoinker#157]
Whenever there is no space left in the terminal for a new table it just cramps everything inside the left over space in terminal previously it was not like this and it automatically used to scroll and made new space in the latest guthub repository chages there has been some issues with the terminal and the load time of the table.
![Screenshot 2023-07-26 144630](https://github.com/zayKenyon/VALORANT-rank-yoinker/assets/69418113/321cfa05-9673-4ed0-a42e-e95642e4af11)

As you can see the issue that im talking about please try to fix it as soon as possible or LMK what changes are reqired to be made Ill make sure to fix them and use it.

**Comment by ThisIsTeddyBear** [comment]
The table which is not being shown is another game and the table was loaded but was not shown in terminal as there was no space left, previous versions of yoinker did not have these issues it something wrong in the newer builds after using RichTable.
Pretty table are a better alternative that was used before I hope you can understand the issue and fix it soon.

**Comment by ThisIsTeddyBear** [comment]
From my testing the feature PREVIOUS_RANK is causing this issue and  leave that alone RICH Table is already very buggy, so I would suggest use some other alternative table representation and also fix the bug in PREVIOUS_RANK which is causing this issue.

**Comment by ThisIsTeddyBear** [comment]
One more importing thing to note is that this issue of table deformation and being limited in the screen only when running the VALORANT-Yoinker with python main.py
After building the application using pip install cx_Freeze and python setup.py build and running the VRY.exe there is no issue in the terminal so Im unaware what is the exact reason causing the issue with tables. Look into this as well once

**Comment by ThisIsTeddyBear** [comment]
@OwOHamper @zayKenyon Can you update on this issue.

**Comment by zayKenyon** [comment]
I think I've fixed it, feel free to re-open if wrong.

**Comment by ThisIsTeddyBear** [comment]
Yup with these changes the issue has been fixed. Hey I would like to know that in anyway can I contribute in this repository ? If there is I would love to make some contributions.

**Comment by zayKenyon** [comment]
Glad to hear it's worked. Thanks for wanting to get involved! A good place to start would be any issues currently tagged with the "help wanted" flair.

**Comment by ThisIsTeddyBear** [comment]
@zayKenyon Sure would see if there is anything that I can help with, One small issue in the table representation is that now the new tables load properly, but the new tables disrupt the old ones but they are created properly no issue in that.

**Comment by zayKenyon** [comment]
Oh crap, what do you mean by disrupt?

**Comment by ThisIsTeddyBear** [comment]
Meaning the new table is created properly but the old table is space which was there it sometimes takes that space and overwrites on top of the old table.

**Comment by ThisIsTeddyBear** [comment]
@zayKenyon As you can see in this image the new tables are propely formed, but now the issue is with this

![Screenshot 2023-08-01 181600](https://github.com/zayKenyon/VALORANT-rank-yoinker/assets/69418113/91fe5699-187b-423f-8857-def37fd01b0e)

Hope you would be able to fix this, just a minor issue.

**Comment by D3CRYPT360** [comment]
Hi am currently working on fixing this, is there anyway i can reach you on discord @ThisIsTeddyBear

**Comment by ThisIsTeddyBear** [comment]
> Hi am currently working on fixing this, is there anyway i can reach you on discord @ThisIsTeddyBear

Here is my discord ID- akuljain

**Comment by ThisIsTeddyBear** [comment]
@zayKenyon and @OwOHamper verified the changes made by @mdanzinger and now this issue has been resolved hence closing this. Thanks for the fix guys 👍 ❤
